python/whylogs/experimental/extras/embedding_metric.py

```diff
@@ -134,10 +134,21 @@ def _update_submetrics(self, submetric: str, data: PreprocessedColumn) -> None:
             self.submetrics[submetric][key].columnar_update(data)
 
     def columnar_update(self, data: PreprocessedColumn) -> OperationResult:
-        if data.numpy.ints is None and data.numpy.floats is None:
+        if data.pandas.tensors is None and data.list.tensors is None:
```

Comment:
Sometimes these are `None` and sometimes an empty list/series.
```suggestion
        if not (data.pandas.tensors or data.list.tensors):
```
I'm not sure if empty `pd.Series` is Falsey. Maybe `if no_data(data.pandas.tensors) and no_data(data.lists.tensors)`, where `no_data()` returns `True` for `None`, `[]`, and empty `pd.Series`?